celery/utils/serialization.py

```diff
@@ -10,18 +10,14 @@
 
 from kombu.utils.encoding import bytes_to_str, str_to_bytes
 
-from celery.five import bytes_if_py2, items, reraise, string_t
-
+from celery.five import bytes_if_py2, items, string_t
```

Comment:
Perhaps out of scope for this PR, but I wonder if we still need `bytes_if_py2` here.